packages/litexa/src/parser/handler.coffee

```diff
@@ -150,12 +150,12 @@ handlerSteps.runConcurrencyLoop = (event, handlerContext) ->
     # work out the language, from the locale, if it exists
     language = 'default'
     if event.request.locale?
-      lang = event.request.locale.toLowerCase()
+      lang = event.request.locale
       langCode = lang[0...2]
```

Comment:
Super minor, but to minimize confusion when reading language/lang/__language/__languages/etc. I'd maybe rename to something like:

```coffeescript
requestLocale = event.request.locale
requestLangCode = requestLocale[0...2]

# Check for corresponding "Litexa language" which can be a locale or a language code
for __language of __languages
  if (__language.toLowerCase() is requestLocale.toLowerCase()) or (__language is requestLangCode)
    language = __language
```